code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -69,7 +69,7 @@
 		var/mob/living/carbon/xenomorph/xeno = locate(href_list["hivemind_jump"])
 		if(!istype(xeno))
 			return
-		var/obj/effect/alien/weeds/nearby_weed = locate() in range("3x3", get_turf(xeno))
+		var/obj/effect/alien/weeds/nearby_weed = locate() in range("6x6", get_turf(xeno)) //increased
```

Comment:
Consider that this kind of comment is not really informative for future coders, as it has no context.

Reply:
fair